tests/test_secp256k1.nim

```diff
@@ -1,10 +1,22 @@
-import ../secp256k1, unittest
+import ../secp256k1, unittest, random
```

Comment:
As you said many times tests are "examples". And if you are using insecure `random` package it means that somebody will start using it too. We should avoid import of `random` at any cost, your `unsafeRng` could generate static data, and produce static private key, but at least we will not show this "unsafe" approach.

Reply:
`random` can be imported but only in a `when isMainModule` section